offchain/danaswap-ui/assets/styles/common.scss

```diff
@@ -0,0 +1,7 @@
+.footer {
+  background: #1F297B;
+}
+
+.button {
+  text-transform: uppercase;
+}
```

Comment:
newline.... are we missing an EditorConfig file??

Reply:
nah, the .editconfig is there, I need to format documents with prettier